# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/animation/AnimationUtils.h
# Commit: 85317d582166
# Full Hash: 85317d5821665149ffa385e8b55b244565c9d4c4
# Author: Tom Tung <ttung@mozilla.com>
# Date: 2020-04-07 21:44:02
# Regressor Bug: 1586761
# File Overlap Count: 1
# Description:
#   Bug 1586761 - P2 - Introduce new TimerPrecisionTypes and a set of new Reduce methods to decide the TimerPrecisionType in the nsRFPService; r=tjr
#   
#   To support checking CrossOriginIsolated in performance.now(), we need to:
#   - Add new types to TimerPrecisionType for nsRFPService
#     - System, HighResAllowed are added
# ==============================================================================

diff -r b51663641450 -r 85317d582166 dom/animation/AnimationUtils.h
--- a/dom/animation/AnimationUtils.h	Wed Mar 25 13:53:46 2020 +0000
+++ b/dom/animation/AnimationUtils.h	Wed Mar 25 14:09:55 2020 +0000
@@ -40,8 +40,8 @@
       // needs to have it's Time Reduction Logic refactored, so it's currently
       // only clamping for RFP mode. RFP mode gives a much lower time precision,
       // so we accept the security leak here for now
-      result.SetValue(nsRFPService::ReduceTimePrecisionAsMSecs(
-          aTime.Value().ToMilliseconds(), 0, TimerPrecisionType::RFPOnly));
+      result.SetValue(nsRFPService::ReduceTimePrecisionAsMSecsRFP(
+          aTime.Value().ToMilliseconds(), 0));
     }
 
     return result;